functions/Invoke-DbaDbShrink.ps1

```diff
@@ -109,7 +109,7 @@ function Invoke-DbaDbShrink {
         Shrinks AdventureWorks2014 to have 50% free space. So let's say AdventureWorks2014 was 1GB and it's using 100MB space. The database free space would be reduced to 50MB.
 
     .EXAMPLE
-        PS C:\> Invoke-DbaDbShrink -SqlInstance sql2014 -Database AdventureWorks2014 -PercentFreeSpace 50 -FileType Data -StepSizeMB 25
+        PS C:\> Invoke-DbaDbShrink -SqlInstance sql2014 -Database AdventureWorks2014 -PercentFreeSpace 50 -FileType Data -StepSize 25
```

Comment:
shouldn't this be .... `-StepSize 25MB` ? (i.e. "throughout the function they'll be bits, but PS has a nice way to use units") ?